mlflow/models/model.py

```diff
@@ -544,11 +544,16 @@ def log(
                             train = df.drop_column("target_label")
                             signature = infer_signature(train, model.predict(train))
 
-        :param input_example: Input example provides one or several examples of
-                              valid model input. The example can be used as a hint of what data to
-                              feed the model. The given example will be converted to a Pandas
-                              DataFrame and then serialized to json using the Pandas split-oriented
-                              format. Bytes are base64-encoded.
+        :param input_example: one or several instances of valid model input. The input example is
```

Comment:
Can a comment be made here to explain the behavior of defining input params in the input_example but leaving the params field undefined at inference time (just to make it clear what the behavior is to users)